libxbrzscale.cpp

```diff
@@ -134,8 +138,30 @@ void displayImage(SDL_Surface* surface, const char* message) {
 }
 */
 
+struct imgUint32{
+  uint32_t *data;
+  int iW;
```

Comment:
why iW / iH? size should be enough, 32x64 is equal to 64x32 in terms of pixel count, since it's a flat pixel array, it doesn't really matter.

Reply:
you are right! :)